terra_notebook_utils/cli/drs.py

```diff
@@ -55,6 +55,22 @@ def drs_copy_batch(args: argparse.Namespace):
     args.workspace, args.google_billing_project = Config.resolve(args.workspace, args.google_billing_project)
     drs.copy_batch(args.drs_urls, args.dst, args.workspace, args.google_billing_project)
 
+@drs_cli.command("head", arguments={
+    "-n": dict(type=int, required=False, default=1, help="Return the first integer n bytes of a file (uncompressed)."),
```

Comment:
Considering matching CLI argument documentation for unix `head`:
```
  -c, --bytes=[-]NUM       print the first NUM bytes of each file;
                             with the leading '-', print all but the last
                             NUM bytes of each file
  -n, --lines=[-]NUM       print the first NUM lines instead of the first 10;
                             with the leading '-', print all but the last
                             NUM lines of each file
```

Reply:
See also https://github.com/DataBiosphere/terra-notebook-utils/pull/107